docs/testing-with-ryujit.md

```diff
@@ -136,7 +175,22 @@ You can selectively disable JIT optimization of a particular function by annotat
 In this case, the annotated `add` function will not be optimized. You can see more detail about `MethodImplAttribute` 
 [here](https://msdn.microsoft.com/en-us/library/system.runtime.compilerservices.methodimplattribute(v=vs.110).aspx).
 
-Note that this method applies to all .NET JIT compilers.
+This is only effective in solving a code generation problem if the incorrect code being generated by the JIT is due to optimization, as opposed to being due to unoptimized code generation.
+
+**Note** This method applies to all .NET JIT compilers.
+
+Disabling optimization of all functions
+=======================================
+
+While troubleshooting, it might be useful to determine if *any* JIT optimization is causing problems. Instead of using the above `MethodImplAttribute` method above, you can set a single environment variable to disable all JIT optimization. Set this variable:
+
+        COMPLUS_JitMinOpts=1
+
+and run your application.
+
+Note that when you run your application under the Visual Studio debugger, it disables JIT optimization by default, to improve the debugging experience. You can read about that [here](https://msdn.microsoft.com/en-us/library/ms241594.aspx).
```

Comment:
This isn't true anymore, "Suppress JIT optimizations on module load" is off by default in 2015.